Ninano/Ninano/Controllers/EventTab/Search/SearchViewController.swift

```diff
@@ -21,7 +21,7 @@ class SearchViewController: UIViewController {
     }
     
     private var articles: APIResponse?
-    private var viewModels = [SearchEventCellViewModel]()
+    private var searchEvent = [SearchEvent]()
     
```

Comment:
```suggestion
    private var searchEvent = [SearchEvent]()
    private var eventList = [Event]()
```
이렇게 바꾸는 건 어떨까요 ? 
배열은 이렇게 list를 붙여서 eventList로 하는 게 더 좋을 거 같구 
Event 라는 이름이 더 잘 어울릴거 같아요 ! (다른 뷰에서도 사용되는 거니까 !)

Reply:
그냥 event로 하려고 하다가 달력 탭에는 별도로 한다고 생각해서 search를 붙였어요! 앱 공통으로 사용하는 것이 맞나요?

Reply:
Event로 반영하였습니다!